Formula/age.rb

```diff
@@ -0,0 +1,18 @@
+class Age < Formula
+  desc "Simple, modern, secure file encryption"
+  homepage "https://filippo.io/age"
+  url "https://github.com/FiloSottile/age/archive/refs/tags/v1.0.0.zip"
+  sha256 "d926766e52a160ed20afd76c5271d182ad4790f7b573e627cd33b44a66510c0d"
+  license "BSD-3-Clause"
+  head "https://github.com/FiloSottile/age.git"
```

Comment:
```suggestion
  head "https://github.com/FiloSottile/age.git", branch: "master"
```

Reply:
Actually, before we put that into a formula, let me change it to `main`.